keps/sig-network/1672-tracking-terminating-endpoints/README.md

```diff
@@ -164,6 +165,12 @@ E2E tests:
 * `EndpointSliceTerminatingCondition` is enabled by default.
 * Consensus on scalability implications resulting from additional EndpointSlice writes with approval from sig-scalability.
 
+#### GA
+
+* E2E tests validating that terminating pods are properly reflected in EndpointSlice API
+* Additional testing / validation on the scalability/performance impact, requiring approval from sig-scalability.
+* All necessary metrics are in place to provide adequate observability and monitoring for this feature.
+
```

Comment:
Couple comments (since I can't comment below);

L208: Were the tests added? If so, can you maybe link them?

L219: IIRC, we were discussing adding some metrics. Has that happened? If not, what is the plan?
[Relying on application-level metrics is not really where we would like to be for such important features like load serving.]

L223: Were the tests run? Can you summarize the findings?

L238: Were metrics added? If so, can you link their names/component exposing them?

Reply:
Btw -- we do have endpointslice metrics now that would be useful here:
 https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/endpointslice/metrics/metrics.go#L30-L53

So users would be able to see the amount of endpoint churn that happens due to this change during upgrade. I'll link these in the KEP too.